ruff_cli/src/cli.rs

```diff
@@ -385,9 +384,7 @@ impl ConfigProcessor for &Overrides {
             config.line_length = Some(*line_length);
         }
         if let Some(max_complexity) = &self.max_complexity {
-            config.mccabe = Some(mccabe::settings::Options {
-                max_complexity: Some(*max_complexity),
-            });
+            config.set_max_complexity(Some(*max_complexity));
```

Comment:
We should probably just remove this. It's sort of incongruous with the other CLI options we provide. But can be a separate change.